src/helm/benchmark/static/schema_safety.yaml

```diff
@@ -106,6 +106,7 @@ perturbations: []
 metric_groups:
   - name: accuracy
     display_name: Accuracy
+    aggregation_strategy: 3
```

Comment:
Adding this as we decided for safety to use this. Local testing confirms this works as expected.